3-enrich/emr-etl-runner/lib/snowplow-emr-etl-runner/emr_job.rb

```diff
@@ -374,6 +374,28 @@ def initialize(debug, enrich, shred, elasticsearch, s3distcp, config, enrichment
           @jobflow.add_step(shred_step)
 
           if s3distcp
+            # Check to see if DashDB target is present, if so add step shreeded types from HDFS -> DashDB targets
+            dashdb_targets = config[:storage][:targets].select {|t| t[:type] == 'dashdb'}
+            if dashdb_targets.length > 0
+              (0..(dashdb_targets.length-1)).each do |i|
```

Comment:
I think the above two lines can be condensed into one - just iterate through the dashdb_targets, it doesn't matter if it's empty or not